src/account.c

```diff
@@ -506,6 +508,31 @@ int account_set_mediaenc(struct account *acc, const char *mencid)
 }
 
 
+/**
+ * Sets audio codecs
+ *
+ * @param acc      User-Agent account
+ * @param codecs   Comma separed list of audio codecs (NULL to disable)
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int account_set_audio_codecs(struct account *acc, const char *codecs)
+{
+	char buf[256];
+	struct pl pl;
+
+	if (!codecs) {
+		list_clear(&acc->aucodecl);
```

Comment:
I believe the list can be cleared in all cases, and then we also dont need
to reset the list element (line 234).

something like this:

```
list_clear()

if (codecs) {
  audio_codecs_decode()
}
```